src/pages/support-commands/index.css

```diff
@@ -0,0 +1,4 @@
+.search-input input {
```

Comment:
Same here, we can use tailwindcss

Reply:
But, input is within the generic TextField component, and .search-input is within the support-commands UI. Hence, we might still need css support here.